tests/kernel/arm_irq_vector_table/Kconfig

```diff
@@ -0,0 +1,4 @@
+config NUM_IRQS
+	int "Nuber of IRQs for this test, made overridable in the .conf file"
+	default 3
+source "$ZEPHYR_BASE/Kconfig.zephyr"
```

Comment:
Should the `default 3` override all the other `default`s on `NUM_IRQS` (from `Kconfig.defconfig` files, etc.)? In that case, the symbol definition needs to go after `source`.

Right now `default 3` will work as a fallback if the symbol has no other defaults (with satisfied conditions). Maybe that's what you want here, but just to check.

Reply:
It really doesn't matter since this is coupled to a `prj.conf` sitting right next to it that sets it to 3

Reply:
Alright, fine then.